docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md

```diff
@@ -14,10 +14,9 @@ Please refer to [Index Documentation](index.md) on details about index.
 
 You can use the `If EXISTS` keywords when dropping tags. These keywords automatically detect if the corresponding tag exists. If it exists, it will be deleted. Otherwise, no tag is deleted.
 
-A vertex can have either only one tag (types) or multiple tags (types).
+A vertex can have one to more tags (types).
 
-In the former case, such a vertex can NOT be accessible after the statement, and edges connected with such vertex may result in DANGLING.
-
-In the latter case, the dropped a vertex is still accessible. But all the properties defined by this dropped tag are not accessible.
+If vertices have only one tag, after dropping the tag, vertices labelled with this tag can NOT be accessible. But the edges connected with the vertices are available.
+If vertices have multiple tags, after dropping one of the tags, the vertices are still accessible. But all the properties defined by this dropped tag are not accessible.
```

Comment:
If a vertex has multiple tags, after one tag is dropped, the vertex is still accessible. 

singularity may make this sentence more understandable.